### PR TITLE
fix: check other items in the array for clearnet

### PR DESCRIPTION
**Pull request by im-adithya**
Currently we only check the first item in the array

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -571,6 +579,8 @@ func (api *API) NewWrappedInvoice(ctx context.Context, request *models.NewWrappe
 
 	api.svc.Logger.Infoln("Requesting own invoice")
 
+	// because we don't want the sender to pay the fee
+	// see: https://docs.voltage.cloud/voltage-lsp#gqBqV
```
Comment (rolznz) [inline api.go:583]: It's just part of the normal wrapped invoice flow. The sender pays the full amount, the receiver pays the fee to get a channel

**Review by rolznz — APPROVED** [review]
utACK

Did this actually happen?

File: api.go (hunk 1 above)
Comment (im-adithya) [reply api.go:583]: It wasn't obvious to me the first time 😅 so thought this would help others while navigating through the code base

**Comment by im-adithya** [comment]
> Did this actually happen?

Luckily both the APIs we use (Voltage and Olympus) have clearnet as first item so isn't an actual issue

**Pull request merged**